x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table.tsx

```diff
@@ -18,9 +18,17 @@ interface Props {
 export const HostsTable: React.FunctionComponent<Props> = ({ nodes }) => {
   const items = useHostTable(nodes);
 
+  const sortSettings = {
```

Comment:
I am not sure which element should be used by default and set the CPU cores but I can change it. The `@timestamp` field is not part of the table so maybe it will be a bit confusing to be used for sorting here (as visually there will be no indication inside the table header which column we use for sorting and the assumption is that it's not sorted), I think memory usage will be a good one. So as we are not sure if we need the default set here I will set `sorting={true}` to be the default and once we decide which column we want we can change it.